docs/Reference/API-Objects.md

```diff
@@ -34,6 +34,17 @@ Returned by [`eth_getBlockByHash`](API-Methods.md#eth_getblockbyhash) and
 | **uncles**           | Array                 | Array of uncle hashes.                                             |
 | **baseFeePerGas**    | Quantity              | The block's [base fee per gas](../Concepts/Transactions/Transaction-Types.md#eip1559-transactions). This field is empty for blocks created before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). |
 
+## Fee history results object
+
+Returned by [`eth_feeHistory`](API-Methods.md#eth_feehistory) for the requested block range.
+If not all blocks are available, only the available subsection of the requested range will be returned.
```

Comment:
Suggestion:
If blocks in the specified block range are not available, then only the fee history for available blocks are returned